sources/packages/web/src/views/aest/student/StudentNotes.vue

```diff
@@ -19,6 +20,7 @@
               :label="item"
               class="p-button-rounded mr-2 secondary-btn-background-lt filter-button"
               :class="{ 'primary-btn-background': filteredNoteType === item }"
+              data-cy="item"
```

Comment:
This one seems to be too generic. I would recommend some more specific.

Reply:
@bhavikccits this is an array of buttons dynamically created. Every button here represents a note type. I can understand without this explanation it is hard to figure out what this is. I would recommend to name it as noteTypeItem.

Reply:
Updated the text with specific words.

Reply:
<img width="430" alt="image" src="https://user-images.githubusercontent.com/98920571/167663674-ef37aa83-faa4-4538-9fa5-a1070956646b.png">